reference/docs-conceptual/community/2020-updates.md

```diff
@@ -1,12 +1,12 @@
 ---
 description: List of changes to the PowerShell documentation for 2020
-ms.date: 05/05/2022
+ms.date: 02/02/2023
 title: What's New in PowerShell Docs for 2020
 ---
 
 # What's new in PowerShell Docs for 2020
 
-This article lists some of the major changes to docs during this previous month and celebrates the
+This article lists notable changes to docs during the previous month and celebrates the
```

Comment:
Nit: Since these are describing the changes over the course of a specific month, I think we might want to reword this sentence. Added to the other documents as well, but they can be rejected together.

```suggestion
This article lists notable changes to docs during the listed month and celebrates the
```

Reply:
I chose a different wording.